MessageApp/Core Layer/NewCoreDataStack.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  NewCoreDataStack.swift
+//  MessageApp
+//
+//  Created by Macbook on 17.10.2022.
+//
+
+import CoreData
+
+protocol ICoreDataStack {
+    func fetch<T>(fetchRequest: NSFetchRequest<T>) -> [T]?
+    func performSave(_ block: @escaping (NSManagedObjectContext) -> Void)
+}
+
+final class NewCoreDataStack: ICoreDataStack {
+    
+    private lazy var container: NSPersistentContainer = {
+       let container = NSPersistentContainer(name: "CoreDataModel")
```

Comment:
Можно в начале файла сделать:
``` Swift
private extension String {
    static let coreDataStackContainerName = "CoreDataModel"
}
```
И использовать это так: 
```suggestion
       let container = NSPersistentContainer(name: String.coreDataStackContainerName)
```
И вообще - со всеми константами, которые используются только внутри файла, можно работать так